http_parser.c

```diff
@@ -2157,6 +2161,7 @@ http_errno_name(enum http_errno err) {
   return http_strerror_tab[err].name;
 }
 
+#ifndef HTTPPARSER_ENABLE_MEMORY_CONSTRAINTS
 const char *
 http_errno_description(enum http_errno err) {
```

Comment:
In the header file, please wrap `HTTPPARSER_ENABLE_MEMORY_CONSTRAINTS` directive where this function is defined.